src/components/MdPortal/MdPortal.test.js

```diff
@@ -32,13 +33,16 @@ test('should render on a custom target', async () => {
     </div>
   `
   await mountTemplate(MdPortal, targetTemplate, {
-    attachToDocument: true
+    attachToDocument: true,
   })
+ 
+  // debugger
+
   const targetEl = document.querySelector('.target')
   const portalWrapper = mount(MdPortal, {
```

Comment:
Fails with `TypeError: Cannot read property 'child' of undefined` in https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/create-wrapper.js#L12